spack.yaml

```diff
@@ -2,9 +2,14 @@
 #
 # It describes a set of packages to be installed, along with
 # configuration settings.
+
+# Build with:
+# - UM7 from dev-access-esm1.6 branch
+# - MOM5 from development branch
+# - CICE4 from ESM1.5 and change to CICE5 once CICE5 SPR has been updated.
 spack:
   specs:
-    - access-esm1p6@git.2024.10.0
+    - access-esm1p6@git.dev_2024.12.0
```

Comment:
This name isn't consistent with the `CALVER.MINOR` versioning format that we have for the other models...is there a reason for that?

Reply:
Yes, it's not to be used for an official release. Is there another way to show that it should not be used for an official release?

Reply:
Nah, thats fine